test/functional/wallet_abandonconflict.py

```diff
@@ -53,7 +54,7 @@ def run_test(self):
         assert_raises_rpc_error(-5, 'Transaction not eligible for abandonment', lambda: alice.abandontransaction(txid=txA))
 
         newbalance = alice.getbalance()
-        assert balance - newbalance < Decimal("0.001")  #no more than fees lost
+        assert_greater_than(Decimal("0.001"), balance - newbalance) #no more than fees lost
```

Comment:
Do we need the `Decimal` here?
Seems basically the same as https://github.com/bitcoin/bitcoin/pull/30019/files?diff=split&w=1#diff-c1987c24fa3e5c788d16de5359500a19a1fea4f379e4288d076cf92be89541dcR478

Reply:
> I see you are a newcomer too.

this isn't necessary...

```Python
assert_greater_than("0.001", balance - newbalance)
```
I wasn't suggesting comparing it to a string, of course, but to a decimal, as in the example I've provided, i.e.:
```Python
assert_greater_than(0.001, balance - newbalance)
```

This seems to be working for me, is it failing for you?

Reply:
Oh, I see. 

Still, better to leave to [preserve potential bugs](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#refactoring) since this is refactoring.